### PR TITLE
[iOS] Update iOS Xcode projects

### PR DESCRIPTION
**Pull request by WeedyWeedSmoker**
## Description

Various maintenance tasks on the iOS Xcode projects…

## Reviewers

@twinaphex

### CODE REVIEW

File: pkg/apple/RetroArch_iOS10.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -81,20 +84,19 @@
 		04E7321822954E99003EB1BC /* GCDWebUploader.bundle */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.plug-in"; path = GCDWebUploader.bundle; sourceTree = "<group>"; };
 		04E7321922954E99003EB1BC /* GCDWebUploader.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = GCDWebUploader.h; sourceTree = "<group>"; };
 		04E7321A22954E99003EB1BC /* GCDWebUploader.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = GCDWebUploader.m; sourceTree = "<group>"; };
-		0FDA2A921BE1AFA800F2B5DA /* RetroArch_iOS9-Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = "RetroArch_iOS9-Info.plist"; path = "/Users/buildbot/buildbot/ios/retroarch/pkg/apple/RetroArch_iOS9-Info.plist"; sourceTree = "<absolute>"; };
+		0FDA2A921BE1AFA800F2B5DA /* RetroArch_iOS9-Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = "RetroArch_iOS9-Info.plist"; path = "/Users/johan/Desktop/RetroArch/pkg/apple/RetroArch_iOS9-Info.plist"; sourceTree = "<absolute>"; };
```
Comment (hizzlekizzle) [inline pkg/apple/RetroArch_iOS10.xcodeproj/project.pbxproj:87]: Is this okay to have the path pointing to (what I guess is) your local path?

Comment (WeedyWeedSmoker) [reply pkg/apple/RetroArch_iOS10.xcodeproj/project.pbxproj]: Fixed… Thanks @hizzlekizzle!

**Comment by WeedyWeedSmoker** [comment]
@twinaphex The change from "libz.dylib" to "libz.tbd" has been made from Xcode 7 (now at version 11…) and iOS 9, but seems to be ok for all iOS versions anyway…

Some file references have long been broken in the different iOS projects…

This is ready to merge!

**Comment by twinaphex** [comment]
Can't we just bake in our own version of libz instead? (deps/libz). That would make it no longer necessary to rely on Apple's own provided version.

**Comment by WeedyWeedSmoker** [comment]
@twinaphex I just made the change so that somewhat recent Xcode would find again all the files and frameworks still in the Xcode projects… I don't think all the files/frameworks are used anyway, the projects will still need some kind of cleaning, but those changes don't really affect compilation, and sadly don't fix RetroArch on iOS 6 which broke recently (I still need to fix my iOS 6 device's screen to get some insight as to why and when it broke…)

**Comment by twinaphex** [comment]
Alright, let's give it a go. I will let you know if there are issues with this.

**Comment by WeedyWeedSmoker** [comment]
@twinaphex Finally found the commit that breaks iOS 6 (and probably iOS 8 too…)

https://github.com/libretro/RetroArch/commit/1d2150cb45f97a96e6b506dc3cf13485c88f2c86

The previous commit work fine with the updated Xcode project for iOS 6 too!

**Comment by twinaphex** [comment]
OK - see if this fixes things for you -

https://github.com/libretro/RetroArch/commit/722f13165b8f82aa32167c9adb16d17d15b2c425

**Comment by WeedyWeedSmoker** [comment]
@twinaphex Fixed it! Working fine on iOS 6 now… Thanks a lot!

**Pull request merged**